openapi.yaml

```diff
@@ -2416,7 +2416,6 @@ paths:
               - type
               allOf:
               - $ref: '#/components/schemas/DomainRecord'
-              - type: object
```

Comment:
I wasn't quite sure what was being gained by adding these `type: objects` in allOfs with only one other $ref. It seems like it's kind of like telling the file to do a union of a set of things (the $ref) and an empty set (`type: object`)--which would just be the set of things?

Reply:
We were only doing it for consistency? If you don't think we need it, then they can be removed. I'm for either case.